test/e2e/toolchainstatus_test.go

```diff
@@ -0,0 +1,94 @@
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	testconfig "github.com/codeready-toolchain/toolchain-common/pkg/test/config"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestForceMetricsSynchronization(t *testing.T) {
+	// given
+	toolchainClusterList := &toolchainv1alpha1.ToolchainClusterList{}
+	ctx, hostAwait, memberAwait, member2Await := WaitForDeployments(t, toolchainClusterList)
+	defer ctx.Cleanup()
+	hostAwait.UpdateToolchainConfig(
+		testconfig.AutomaticApproval().Enabled(),
+		testconfig.Metrics().ForceSynchronization(false))
+
+	userSignups := CreateMultipleSignups(t, ctx, hostAwait, memberAwait, 2)
+
+	// delete the current toolchainstatus/toolchain-status resource and restart the host-operator pod,
+	// so we can start with accurate counters/metrics (and not get flaky because of previous tests)
+	err := hostAwait.DeleteToolchainStatus("toolchain-status")
+	require.NoError(t, err)
+	err = hostAwait.DeletePods(client.InNamespace(hostAwait.Namespace), client.MatchingLabels{"name": "host-operator"})
+	require.NoError(t, err)
+
+	metricsAssertion := InitMetricsAssertion(t, hostAwait, []string{memberAwait.ClusterName, member2Await.ClusterName})
+
+	t.Run("verify metrics are still correct after restarting pod", func(t *testing.T) {
+		// when
+		err := hostAwait.DeletePods(client.InNamespace(hostAwait.Namespace), client.MatchingLabels{"name": "host-operator"})
```

Comment:
May I know what is the purpose of this deletion of the host-operator pod?

Reply:
that's a bit superfluous and unnecessary, indeed. I removed it in cff0582